tests/emodulus_calibration/test_emodulus_calibration.py

```diff
@@ -0,0 +1,31 @@
+from lebedigital.calibration.calibrationWorkflow import esimate_Youngs_modulus
+from lebedigital.calibration.utils import query_KG
+from probeye.ontology.knowledge_graph_import import import_parameter_samples
+
+import pytest
+import numpy as np
+import os
+from pathlib import Path
+
+def test_emodulus_calibration():
+
+    # query knowledge graph. Note KG team needs to provide this.
+    print("Calibration test started")
+    output = query_KG(
```

Comment:
why do you have to query a KG for a simple CSV file? The tests should be unit tests, thus not include other complex tasks. In your case, I would expect the input to be a load displacement curve, the output a posterior of E

Reply:
This is outdated after our discussion.